tests/test_query.py

```diff
@@ -550,6 +550,30 @@ def test_casefold(self):
 
     self.assertEqual(self._q(query.casefold(u'\u212B', "NFKCCaseFold")), u'\u00E5')
 
+
+  def test_starts_with(self):
+    self.assertEqual(self._q(query.starts_with("faunadb", "fauna")), True)
+    self.assertEqual(self._q(query.starts_with("faunadb", "F")), False)
+
+  def test_ends_with(self):
+    self.assertEqual(self._q(query.ends_with("faunadb", "fauna")), False)
+    self.assertEqual(self._q(query.starts_with("faunadb", "")), True)
```

Comment:
copy and pasta here?

btw, I personally don't like this test: `string ends with empty-string` , could we change to something like `query.ends_with("faunadb", "db"), True` ?